README.asciidoc

```diff
@@ -30,7 +30,13 @@ a| image::images/raspberry_pi_os_lite.png[64bit Lite,200]
 1. Enable passwordless ssh, by copying your public key to the raspberry pi's `~/.ssh/authorized_keys` file. For convenience, you can also use `ssh-copy-id -i $PATH_TO_KEY pi@$PI_IP_ADDRESS`
 1. Ensure you have added the key to your agent with `ssh-add -K $PATH_TO_KEY`. Now try to connect via SSH and ensure you don't get a password prompt.
 
+== Preparations
+The first thing you'll need for the cluster setup is ansible. So please install ansible following the guide here: https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible Installation Guide]
 
+You'll also need git cli in order to checkout the project and run k3s-install (see below). 
+Something like `sudo apt install git-all` will do the job.
```

Comment:
Is this something you had to do during the install on the raspberry node? 

I didn't need this, so I wonder if we started from a different SD card image perhaps? 🤔

Reply:
Yes exactly. Didn't you install git at all or it was already installed?

Reply:
it was already installed. 

In that case, let's change the wording slightly, to cover both cases: 
```suggestion
In case it isn't already installed (try `git --version`), something like `sudo apt install git-all` will do the job.
```